commit 8990186b15923096054d42de03cbbab9453730f3
Author: Joerg Bornemann <joerg.bornemann@digia.com>
Date:   Mon Dec 17 10:06:38 2012 +0100

    fix duplicate TRANSLATION entry in generated RC file
    
    Task-number: QTBUG-28625
    
    Change-Id: I9bd6098c7557ca008211acef172f5c51a6766998
    Reviewed-by: Oswald Buddenhagen <oswald.buddenhagen@digia.com>
    (cherry picked from commit 3e94e8daf36e90e73ca475a81c9b9a3af40b45b3)

diff --git a/qmake/generators/win32/winmakefile.cpp b/qmake/generators/win32/winmakefile.cpp
index 0d855a3..bf03bdc 100644
--- a/qmake/generators/win32/winmakefile.cpp
+++ b/qmake/generators/win32/winmakefile.cpp
@@ -466,10 +466,6 @@ void Win32MakefileGenerator::processRcFileVar()
            << QString("0x%1").arg(rcLang, 4, 16, QLatin1Char('0'))
            << ", " << QString("%1").arg(rcCodePage, 4) << endl;
         ts << "\t\tEND" << endl;
-        ts << "\t\tBLOCK \"VarFileInfo\"" << endl;
-        ts << "\t\tBEGIN" << endl;
-        ts << "\t\t\tVALUE \"Translation\", 0x409, 1200" << endl;
-        ts << "\t\tEND" << endl;
         ts << "\tEND" << endl;
         ts << "/* End of Version info */" << endl;
         ts << endl;
